gruntfile.js

```diff
@@ -55,6 +55,21 @@ module.exports = function (grunt) {
   grunt.registerTask('run-dev', ['build', 'shell:run', 'watch:build']);
   grunt.registerTask('clear', ['clean:app']);
 
+  // Load content of `vendor.yml` to src.jsVendor, src.cssVendor and src.angularVendor
+  grunt.registerTask('vendor', 'vendor:<minified|regular>', function(min) {
+    // Argument `min` defaults to 'minified'
+    var m = ( min === '' ) ? 'minified' : min;
+    var v = grunt.file.readYAML('vendor.yml');
+    for (var type in v) { if (v.hasOwnProperty(type)) { var z = v[type][m];
```

Comment:
Mind rewriting the code to make it a bit more readable? 

* No point in having multiple statements on the same line (for / if / if-content)
* Use understandable variable names

Told you that compact != readable and I'd favor the second.